docs/sources/developers/plugins/add-query-editor-help.md

```diff
@@ -0,0 +1,72 @@
+## Add a query editor help component
+
+By adding a help component to your plugin, you can for example create "cheat sheets" with commonly used queries. When the user clicks on one of the examples, it automatically updates the query editor. It's a great way to increase productivity for your users.
+
+1. Create a file `QueryEditorHelp.tsx` in the `src` directory of your plugin, with the following content:
+
+   ```ts
+   import React from 'react';
+   import { QueryEditorHelpProps } from '@grafana/data';
+
+   export default (props: QueryEditorHelpProps) => {
+     return (
+       <h2>My cheat sheet</h2>
+     );
+   };
+   ```
+
+2. Configure the plugin to use the `QueryEditorHelp`.
```

Comment:
If possible, use repetitive numbering per the [Grafana markdown guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-markdown-guide.md#numbered-lists).